src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  const LONG_TERM = 7;
+  const MID_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const MID_TERM_DISCOUNT = 20;
+  let amount = 0;
+
+  if (days < MID_TERM) {
+    amount = rent * days;
+  }
+
+  if (days >= MID_TERM && days < LONG_TERM) {
+    amount = rent * days - MID_TERM_DISCOUNT;
+  }
+
+  if (days >= LONG_TERM) {
+    amount = rent * days - LONG_TERM_DISCOUNT;
+  }
```

Comment:
The condition `days >= LONG_TERM` correctly applies the long-term discount. This logic is correct and follows the task requirements.